### PR TITLE
Adds missing field due to new opensearch version

### PR DESCRIPTION
**Pull request by Jakob3xD**
### Description
Fix current integration tests due to missing fields.

### Issues Resolved
_List any issues this PR will resolve, e.g. Closes [...]._ 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,10 +6,12 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
 ### Added
 - Adds the `Routing` field in SearchHit interface. ([#516](https://github.com/opensearch-project/opensearch-go/pull/516))
-
 - Adds the `SearchPipelines` field to `SearchParams` ([#532](https://github.com/opensearch-project/opensearch-go/pull/532))
+- Adds `search_idle_reactivate_count_total` field to structs ([#552](https://github.com/opensearch-project/opensearch-go/pull/552))
```
Comment (dblock) [inline CHANGELOG.md:10]: This is really "Added support for OpenSearch 2.14", isn't it?

**Review by dblock — DISMISSED** [review]
I think the CHANGELOG should just say "Added support for OpenSearch 2.14", although mentioning fields is fine too. 

There's a tab/space change here (https://github.com/opensearch-project/opensearch-go/pull/552/files?w=1 shows a clean diff), is this not desired or are we correcting poor auto-format from before?

Feel free to amend or just merge.

**Comment by Jakob3xD** [comment]
> There's a tab/space change here (https://github.com/opensearch-project/opensearch-go/pull/552/files?w=1 shows a clean diff), is this not desired or are we correcting poor auto-format from before?

Correcting poor format as the new line between the messages is unneded.

**Comment by dblock** [comment]
> > There's a tab/space change here (https://github.com/opensearch-project/opensearch-go/pull/552/files?w=1 shows a clean diff), is this not desired or are we correcting poor auto-format from before?
> 
> Correcting poor format as the new line between the messages is unneded.

I didn't mean CHANGELOG, in all the .go files there's a diff spacing.

**Comment by Jakob3xD** [comment]
> I didn't mean CHANGELOG, in all the .go files there's a diff spacing.

The Spacing is done automatically by go fmt as the new struct field name is so long.

**Review by dblock — APPROVED** [review]

**Pull request merged**